### PR TITLE
Fixed HMI Uncaught SyntaxError after processing SetDisplayLayout

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
Fixes [#609](https://github.com/smartdevicelink/sdl_hmi/issues/609)

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
A `DEFAULT` available template is removed from the display capabilities because it's not defined on the HMI.
The expected result of the [issue-609](https://github.com/smartdevicelink/sdl_hmi/issues/609) must be replaced with an error result code.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by ValeriiMalkov** [comment]
Hi @theresalech This is a regression issue and PR is ready for Livio team review

**Comment by ShobhitAd** [comment]
@ValeriiMalkov I'm not sure that removing "DEFAULT" from the available templates is the best solution. If `displayLayout ` "DEFAULT" is sent in the `SetDisplayLayout` request, the HMI should select an appropriate template name based on the App HMI type or default to one (there is already a case for this  https://github.com/smartdevicelink/sdl_hmi/blob/release/5.6.0/ffw/UIRPC.js#L517).

The reported error is caused by changes made in #590. https://github.com/smartdevicelink/sdl_hmi/blob/release/5.6.0/app/controller/sdl/Abstract/Controller.js#L1656 gets the template capabilities based on the template name (`appModel.templateConfiguration.template`). `appModel.templateConfiguration.template` is set to DEFAULT for this case and since there are no template capabilities for "DEFAULT the lookup doesn't return a valid response.

I think for fix we might have to select the appropriate template name (as done in https://github.com/smartdevicelink/sdl_hmi/blob/release/5.6.0/ffw/UIRPC.js#L517) in the case that `appModel.templateConfiguration.template` is set to DEFAULT.

File: app/model/sdl/Abstract/Model.js [hunk 1]
```diff
@@ -628,7 +628,6 @@ SDL.SDLModel = Em.Object.extend({
         return model.appType.indexOf('WEB_VIEW') >= 0;
       }
       case 'NON-MEDIA':
```
[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply app/model/sdl/Abstract/Model.js]: Sorry, my mistake. I forgot to re-add it

[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply app/model/sdl/Abstract/Model.js]: Changes in commit 2048ee316e10e70360be939ee653f933a3e18979

File: app/model/sdl/Abstract/data.js [hunk 2]
```diff
@@ -1183,7 +1183,7 @@ SDL.SDLModelData = Em.Object.create(
               ],
               "imageTypeSupported": ["STATIC", "DYNAMIC"],
               "numCustomPresetsAvailable": 8,
-              "templatesAvailable": ["MEDIA", "NON-MEDIA", "DEFAULT", "NAV_FULLSCREEN_MAP", 'WEB_VIEW'],
+              "templatesAvailable": ["MEDIA", "NON-MEDIA", "NAV_FULLSCREEN_MAP", 'WEB_VIEW'],
```
[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply app/model/sdl/Abstract/data.js]: Changes in commit 2048ee316e10e70360be939ee653f933a3e18979

File: app/controller/sdl/Abstract/Controller.js [hunk 3]
```diff
@@ -1653,7 +1653,20 @@ SDL.SDLController = Em.Object.extend(
       const windowType = (windowID === undefined || windowID === 0) ? "MAIN" : "WIDGET";
       const windowTypeSupported = SDL.SDLModelData.defaultWindowCapability[windowType].systemCapability.displayCapabilities[0].windowTypeSupported;
 
-      let templateCapabilities = SDL.deepCopy(SDL.templateCapabilities[appModel.templateConfiguration.template])
+      let template = appModel.templateConfiguration.template;
+      if(template === 'DEFAULT') {
+        if(appModel.appType.includes('NAVIGATION')) {
+          template = 'NAV_FULLSCREEN_MAP';
+        } else if(appModel.appType.includes('WEB_VIEW')) {
+          template = 'WEB_VIEW';
```
[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply app/controller/sdl/Abstract/Controller.js]: Changes in commit 2048ee316e10e70360be939ee653f933a3e18979

**Review by ShobhitAd — APPROVED** [review]

**Pull request merged**